huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_address_group_test.go

```diff
@@ -95,16 +93,289 @@ func TestAccIpAddressGroup_basic(t *testing.T) {
 	})
 }
 
-func testIpAddressGroup_basic(name, description string) string {
+func testIpAddressGroup_basic_step_1(name string) string {
 	return fmt.Sprintf(`
 resource "huaweicloud_ga_address_group" "test" {
   name        = "%[1]s"
-  description = "%[2]s"
+  description = "Created by terraform"
+
+  ip_addresses {
+    cidr        = "192.168.1.0/24"
+    description = "The first CIDR block in the IP address group"
+  }
+
+  ip_addresses {
+    cidr        = "192.168.2.0/24"
+    description = "The second CIDR block in the IP address group"
+  }
+}
+`, name)
+}
+
+func testIpAddressGroup_basic_step_2(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_ga_address_group" "test" {
+  name = "%[1]s"
+
+  ip_addresses {
```

Comment:
suggest to use `dynamic` block, for example
```hcl
  dynamic "ip_addresses" {
    for_each = range(0, 10)
    content {
      cidr        = "192.168.${ip_addresses.value}.0/24"
      description = "The ${ip_addresses.value}th IP addresses included in the address group"
    }
  }
```

Reply:
done